certbot/certbot/_internal/main.py

```diff
@@ -1537,6 +1537,12 @@ def certonly(config: configuration.NamespaceConfig, plugins: plugins_disco.Plugi
 
     lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
 
+    # If a new cert was issued and we were passed an installer, we can safely
+    # run `installer.restart()` to load the newly issued certificate
+    if lineage and installer and not config.dry_run:
+        display_util.notify(f"Reloading {config.installer} server after certificate issuance")
```

Comment:
What do you think about demoting this message to `logger.info`?

Certbot used to show a lot more of these kinds of status messages, and this is where we landed with deciding what to show to the user by default (comment from `log.py`):

>The `logging` module is useful for recording messages about about what
Certbot is doing under the hood, but do not necessarily need to be shown
to the user on the terminal.

Reply:
Sounds good to me, especially if reloading the installer is expected behavior at this point.